tests/test_infer/test_kvcache_manager.py

```diff
@@ -0,0 +1,165 @@
+import random
+from dataclasses import dataclass
+
+import torch
+from transformers.models.llama import LlamaConfig
+
+from colossalai.inference.kv_cache import CacheBlock, KVCacheManager
+from colossalai.logging import disable_existing_loggers
+from colossalai.testing import parameterize
+
+
+@dataclass
+class SampleInferenceConfig:
```

Comment:
This class and the `InferenceConfig` look confusing to me

Reply:
`SampleInferenceConfig ` class was removed and pytest is using `InferenceConfig` at this moment.